SPECS/csi-driver-lvm/csi-driver-lvm.spec

```diff
@@ -0,0 +1,41 @@
+Summary:        Container storage interface for logical volume management
+Name:           csi-driver-lvm
+Version:        0.4.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/metal-stack/csi-driver-lvm
+#Source0:       https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}-govendorcached.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated GO dependencies from this tarball, since network is disabled during build time.
+#   1. wget https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2022-09-18 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-govendorcached.tar.gz .
```

Comment:
This looks like we're packing both the cache and the project sources into one tarball. For the sake of consistency, I'd suggest splitting the two, like we do in similar Go projects (`cni.spec`, `cf-cli.spec`, `kured.spec`, etc.) but I don't have any strong opinions here. Treat this as a polite and mild request.;)

Reply:
I see, the question is if I put the source code as SOURCE0 while the cache one as SOURCE1, how can I make sure when building it only takes SOURCE1 to build? Cuz SOURCE0 may failed due to network disabled.

Reply:
I have revised a new commit that split it to two, source0 will be the original one, while source1 will be the one with cached vendor & tidy.

Reply:
Resolved.